di-ipv-credential-issuer-stub/src/main/java/uk/gov/di/ipv/stub/cred/handlers/AuthorizeHandler.java

```diff
@@ -260,278 +257,164 @@ public AuthorizeHandler(
                 return viewHelper.render(frontendParams, "authorize.mustache");
             };
 
-    public Route generateResponse =
-            (Request request, Response response) -> {
-                QueryParamsMap queryParamsMap = request.queryMap();
-
-                AuthorizationErrorResponse requestedAuthErrorResponse =
-                        handleRequestedError(queryParamsMap);
-                if (requestedAuthErrorResponse != null) {
-                    response.redirect(requestedAuthErrorResponse.toURI().toString());
-                    return null;
-                }
-
-                String clientIdValue = queryParamsMap.value(RequestParamConstants.CLIENT_ID);
-                String requestValue = queryParamsMap.value(RequestParamConstants.REQUEST);
-
-                ClientConfig clientConfig = ConfigService.getClientConfig(clientIdValue);
+    public final Route apiAuthorize =
+            (Request request, Response response) ->
+                    generateResponse(
+                            OBJECT_MAPPER.readValue(request.body(), AuthRequest.class), response);
```

Comment:
Do we need to validate the JAR here?

Reply:
I don't think so. We'd just be validating that the thing we're passing in ourselves is correct. And we go on to decrypy the jar, and then extract the signed jwt. And then pull out all the details from there. If anything is wrong it'll blow up at some point and we'd see some failing tests.